src/components/ChallengeList/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { data } from "./data";
+
+type Task = {
+  task: string;
+};
+
+type Challenge = {
+  title: string;
+  points: number;
+  tasks: Task[];
+  winner: string;
+  winnerPoints: number;
+};
+
+export const ChallengeList: React.FC = () => {
+  return (
+    <section className="my-10 grid grid-cols-1 gap-y-6 gap-x-5 dark:text-white md:grid-cols-2 lg:grid-cols-3">
+      {data.map(({ title, points, tasks, winner, winnerPoints }: Challenge) => (
```

Comment:
celkovo je to ale podla mna zbytocne to tu otypovavat...TS si to sam inferne
a ked tak by som otypoval tie data hore a nie tu tomu davat typ

Reply:
true